src/myTestEndpoint.js

```diff
@@ -71,6 +71,7 @@ const myEndpoint = preparator(
               description: "Some array item text",
               keys: {
                 a: { type: "int" },
+                c: { type: "object", optional: true },
```

Comment:
The tests fail exactly as they should:

![image](https://user-images.githubusercontent.com/16835755/166922869-e9631d4d-03c0-4408-9577-d661954d1c80.png)

the line you added is not a valid object-type. The keys are missing